src/components/FeaturedProduct.jsx

```diff
@@ -0,0 +1,97 @@
+import mrfBat from "../assets/mrf-bat.png"
+import nikeFootball from "../assets/nike-football.png"
+import kookaburraBall from "../assets/kookabura-ball.png"
+import nikeShoes from "../assets/nike-shoes.png"
```

Comment:
you can create an index.js file in assets folder make an object and can import all images at once...i

Reply:
Good point, will surely refactor it